marshmallow_recipe/fields.py

```diff
@@ -579,10 +589,16 @@ def _serialize(self, value: Any, attr: Any, obj: Any, **kwargs: Any) -> Any:
 
         def _deserialize(self, value: Any, attr: Any, data: Any, **kwargs: Any) -> Any:
             result = super()._deserialize(value, attr, data, **kwargs)
-            if self.strip_whitespaces and result is not None:
-                result = result.strip()
+            if result is not None:
+                if self.strip_whitespaces:
+                    result = result.strip()
+
                 if self.allow_none and len(result) == 0:
```

Comment:
different behaviour in v2 (correct) and v3 (incorrect)

Reply:
Fixed